packages/flutter/lib/src/rendering/error.dart

```diff
@@ -79,14 +81,15 @@ class RenderErrorBox extends RenderBox {
 
   /// The text style to use when painting [RenderErrorBox] objects.
   static ui.TextStyle textStyle = new ui.TextStyle(
-    color: const Color(0xFFFFFF00),
+    color: const Color(0xFFFFFF66),
```

Comment:
Can we make the background darker and the foreground brighter? White on maroon, say? Or yellow on maroon?